lib/rubocop/cop/rails/rake_environment.rb

```diff
@@ -3,16 +3,15 @@
 module RuboCop
   module Cop
     module Rails
-      # This cop checks rake task definition without `environment` dependency.
-      # `environment` dependency is important because it loads application code
-      # for the rake task. The rake task cannot use application code, such as
-      # models, without `environment` dependency.
+      # This cop checks for Rake tasks without the `environment` dependency. The
```

Comment:
I think that the document will be more consistent with `:environment` task.

```diff
-This cop checks for Rake tasks without the `environment` dependency. The
+This cop checks for Rake tasks without the `:environment` task dependency. The
```

Reply:
Great idea. I made the change